app/_config/mimevalidator.yml

```diff
@@ -0,0 +1,9 @@
+---
+Name: app-mimeuploadvalidator
+After: '#mimeuploadvalidator'
+Only:
+  moduleexists: 'silverstripe/mimevalidator'
+---
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\Assets\Upload_Validator:
+    class: SilverStripe\MimeValidator\MimeUploadValidator
```

Comment:
New line at end of file :)

Reply:
That just gets copied over when you install recipe-core 4.6 now.